app/controllers/ops_controller.rb

```diff
@@ -67,7 +67,6 @@ def display_ae_namespaces
     'fetch_audit_log'           => :fetch_audit_log,
     'fetch_log'                 => :fetch_log,
     'fetch_production_log'      => :fetch_production_log,
-    'log_depot_edit'            => :log_depot_edit,
```

Comment:
Once this goes, I think all (most?) of the following can go too:

```
$ git grep log_depot_edit
app/controllers/ops_controller.rb:70:    'log_depot_edit'            => :log_depot_edit,
app/controllers/ops_controller.rb:111:    'zone_log_depot_edit'       => :log_depot_edit,
app/controllers/ops_controller.rb:474:        action_url = "log_depot_edit"
app/controllers/ops_controller.rb:628:    elsif nodetype == "log_depot_edit"
app/controllers/ops_controller.rb:794:        if ["log_depot_edit", "ze"].include?(nodetype)
app/controllers/ops_controller/diagnostics.rb:77:  def log_depot_edit
app/controllers/ops_controller/diagnostics.rb:78:    assert_privileges("#{@sb[:selected_typ] == "miq_server" ? "" : "zone_"}log_depot_edit")
app/controllers/ops_controller/diagnostics.rb:147:      replace_right_cell(:nodetype => "log_depot_edit")
app/controllers/ops_controller/diagnostics.rb:280:    assert_privileges("#{@sb[:selected_typ] == "miq_server" ? "" : "zone_"}log_depot_edit")
app/helpers/application_helper/toolbar/diagnostics_server_center.rb:83:      :log_depot_edit,
app/helpers/application_helper/toolbar/diagnostics_zone_center.rb:91:      :zone_log_depot_edit,
app/views/layouts/angular-bootstrap/_auth_credentials_angular_bootstrap.html.haml:2:- validate_url           ||= 'log_depot_edit'
app/views/layouts/angular/_auth_service_account_angular.html.haml:2:- validate_url ||= 'log_depot_edit'
app/views/ops/_log_collection.html.haml:5:                     'save-url'                       => "/#{controller_name}/log_depot_edit/",
config/routes.rb:2400:        log_depot_edit
spec/config/routes.pending.yml:887:- log_depot_edit
spec/controllers/ops_controller/diagnostics_spec.rb:322:    context "#log_depot_edit" do
spec/controllers/ops_controller/diagnostics_spec.rb:342:        controller.send(:log_depot_edit)
spec/controllers/ops_controller_spec.rb:294:      post :x_button, :params => {:id => @miq_server.id, :pressed => 'log_depot_edit', :format => :js}
spec/controllers/ops_controller_spec.rb:301:      post :x_button, :params => {:id => @miq_server.id, :pressed => 'log_depot_edit', :button => "cancel", :format => :js}
spec/controllers/ops_controller_spec.rb:308:      post :x_button, :params => {:id => @miq_server.id, :pressed => 'log_depot_edit', :button => "save", :format => :js}
spec/javascripts/components/ops/log_collection_form_spec.js:20:      saveUrl: "/ops/log_depot_edit/",
spec/javascripts/components/ops/log_collection_form_spec.js:68:      expect(miqService.miqAjaxButton).toHaveBeenCalledWith("/ops/log_depot_edit/123456?button=cancel", true);
spec/javascripts/components/ops/log_collection_form_spec.js:78:      var url = "/ops/log_depot_edit/123456?button=save";
spec/routing/ops_routing_spec.rb:53:    log_depot_edit
```

Reply:
yeah i've already removed most of those on my local just testing that nothings broken will push once done